projects/blindex/index.js

```diff
@@ -72,7 +73,10 @@ async function getBDStableCollateralBalances(block, chainName, bdstable) {
   for (let index = 0; index < bdstableCollateralPools.length; index++) {
     const collateralAddress = await (
       await sdk.api.abi.call({
-        target: bdstableCollateralPools[index],
+        target: formatAddressChecksum(
```

Comment:
I didn't check this sdk, are you sure you need to lower case here?

Reply:
Yep, testedwithout it and it failed